Change the explanation and example for splice() function in javascript challenges section (#36589)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-items-using-splice.english.md

## Description
<section id='description'>

> Remember in the last challenge we mentioned that <code>splice()</code> can take up to three parameters? Well, in addition to removing elements with the first two parameters, we can use that third parameter, comprised of one or more element(s), to add them as well. This can be incredibly useful for quickly switching out an element, or a set of elements, for another.

Reviewer: ```suggestion
Remember in the last challenge we mentioned that <code>splice()</code> can take up to three parameters? Well, you can use the third parameter, comprised of one or more element(s), to add to the array. This can be incredibly useful for quickly switching out an element, or a set of elements, for another.
```
The first formulation of the sentence was a little complicated.  I think it should be focused on what's new, namely the third parameter and what it does.

```js
const numbers = [10, 11, 12, 12, 15]
const startIndex = 3;
const amountToDelete = 1;

numbers.splice(startIndex, amountToDelete, 13, 14)
// the second entry of 12 is removed, and we add 13 and 14 at the same index

console.log(numbers);
// returns [ 10, 11, 12, 13, 14, 15 ]
```

> Here we begin with an array of numerical values. Using <code>splice()</code>, we have taken an index at which to begin removing elements (3), the number of elements to be removed (1), and the values (13, 14) to be inserted at that same index. Note that the values passed in the third parameter are separated by commas, but require no additional formatting.

Reviewer: ```suggestion
Here we begin with an array of numbers. We then pass the following to <code>splice()</code>. The index at which to begin deleting elements (3), the number of elements to be deleted (1), and the elements (13, 14) to be inserted at that same index. Note that there can be any number of elements (separated by commas) following <code>amountToDelete</code>, each of which gets inserted.
```
This is tricky to word correctly!  However, I think this gets the point across without blurring the lines between argument and parameter too much.  Also, I think we should stick with with 'element' and 'delete' rather than switching in 'values' and 'remove'.  Splice is annoying enough, we want to make the description as simple as we can.

Author: Yea, your changes definitely make the description simple.
